internal/service/advancedclustertpf/data_source.go

```diff
@@ -14,6 +14,12 @@ import (
 var _ datasource.DataSource = &ds{}
 var _ datasource.DataSourceWithConfigure = &ds{}
 
+const (
+	errorReadDatasource                 = "Error reading  advanced cluster datasource"
+	errorReadDatasourceAsymmetric       = "Error reading  advanced cluster datasource"
+	errorReadDatasourceAsymmetricDetail = "Cluster name %s. Please add `use_replication_spec_per_shard = true` to your data source configuration to enable asymmetric shard support. Refer to documentation for more details."
```

Comment:
Could you provide a link to the doc as part of the error message? We do this in the CLI. You should ask the doc team for a dochub link if the documentation is in the MDB website

Reply:
Good point. Added the link to our migration guide that has more details in 1c35df47ce5467323b0ec5d37c32246baff14536